inference_rules.adoc

```diff
@@ -428,6 +428,79 @@ division allows calibration for quantization and does not allow any retraining.
 The unqualified name “MLPerf” must be used when referring to a Closed Division
 suite result, e.g. “a MLPerf result of 4.5.”
 
+=== Network Division
+
+The Network division inherits all requirements from the Closed division and imposes further constraints and supports only the Datacenter suite.
+
+In the Network division the SUT is connected to the loadgen over a network fabric. The Query Dispatch Library (QDL) component which is the client-side SUT proxy which connects the Loadgen to an SUT over a network. The QDL is a submitter implemented component.
+
+==== QDL Constraints
+
+* The QDL is not allowed to do any pre-processing.
+* If an SUT compresses its output, the QDL must decompress the output. Decompression is a timed operation. No other post-processing in the QDL is allowed.
+* The QDL is not allowed to batch queries.
+* The Name method's return value must contain the substring "Network SUT".
+* The Name method's implementation must include at least one round trip over the network. The Name method must not return until the round trip is complete.
+
+==== General Constraints
+
+MLPerf distinguishes between fabric interconnects and bus interconnects. Fabrics interconnects are required and bus interconnects are forbidden.
+
+A fabric interconnect must:
+
+1. Work as out-of-the-box chassis-to-chassis interconnect
+1. Use wireless, copper, or fiber-optics media
+1. Connect systems in meters to kilometers distances
+1. Use switch topology
+1. Be highly scalable, reliable, and fault-tolerant
+
+Examples of forbidden bus interconnect include: PCIe/CXL/CCIX, Hypertransport, NVLink, QPI, UPI, and ICI (interchip interconnect).
+
+The SUT and QDL must run on two physically separated and distinct systems.
+
+Fabric and protocol must be reported in the submission metadata. Submission metadata must be include sufficient to determine OSI layers one through four of the submission's network stack.
+
+==== SUT over the Network and Setup Constraints
+
+* SUT parameters and configuration must be specified in detail in the submission results. 
+* Network interface characteristics must be added to the report, both at the SUT side and at the loadgen server side, including networking components or any other hardware in the system that either stands between them or are involved in any way in the test.
```

Comment:
We need to be more specific than "any other hardware that ... is involved in any way in the test"

Reply:
...and name any other components through which data passes between loadgen and the SUT for reproducibility.

changed "be specified in detail"  to "uniquely and specifically named"